README.asciidoc

```diff
@@ -304,22 +304,52 @@ Please let us know if you have any ideas about how to make the default config mo
 ==== Server-specific configuration
 
 Many servers accept configuration options that are not part of the LSP spec.  The TOML table
-`[language.<filetype>.settings]` holds those configuration options.  It has the same structure
+`[language_server.<server_name>.settings]` holds those configuration options.  It has the same structure
 as the corresponding fragments from VSCode's `settings.json`. For example:
 
 [source,toml]
 ----
-[language.go]
+[language_server.gopls]
 ...
-settings_section = "gopls"
-[language.go.settings.gopls]
+[language_server.gopls.settings.gopls]
 "formatting.gofumpt" = true
 ----
 
-During server initialization, kak-lsp sends the section specified by `settings_section`; in this
-case `{"formatting.gofumpt":true}`.  Whenever you change the Kakoune option `lsp_config`, the
-same section is sent via `workspace/didChangeConfiguration`.  Additionally, kak-lsp will send
-arbitrary sections that are requested by the server in `workspace/configuration`.
+During server initialization, kak-lsp sends either the section specified by `settings_section`
+or the section with same name as the server; in this case `{"formatting.gofumpt":true}`.
```

Comment:
> or the section with same name as the server

Hmmm, really? IIRC `settings_section` has no default value.
If I'm correct, then we need to add a `settings_section` to the `tailwindcss` example.
(Unfortunately the whole dynamic settings passing is super weird due to the lack of standardization in LSP.)

Reply:
ok I think you're right, `settings_section` is optional